executor/simple.go

```diff
@@ -111,6 +136,15 @@ func (e *baseExecutor) releaseSysSession(ctx context.Context, sctx sessionctx.Co
 	sysSessionPool.Put(sctx.(pools.Resource))
 }
 
+// Since the environment variables in the session are changed, closing the session does not return the session
```

Comment:
Please rewrite the comment to the style like: 
```
// clearSysSession ...
```
Refer to https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
done